compiler/optimizer/ValuePropagationCommon.cpp

```diff
@@ -922,6 +922,47 @@ bool OMR::ValuePropagation::transformUnsafeCopyMemoryCall(TR::Node *arrayCopyNod
    return false;
    }
 
+static void setNodeOnList(TR::Node *currNode, TR_BitVector *nodeList)
```

Comment:
May I ask you to add some Doxygen comments or at least some description of what it's doing?  I know there's a bit of a description that appears where it's called, but I think it would help to have it described here.